app/dashboard/helpers.py

```diff
@@ -336,6 +323,27 @@ def process_bounty_details(bountydetails, url, contract_address, network):
             for interested in last_bounty.interested.all():
                 new_bounty.interested.add(interested)
 
+        if fments:
```

Comment:
thanks for handling this!   i admit the current `current_bounty` stuff is kind of an uncommon pattern...

i wonder if we might want to move lines 326 => 336 into a sub function.. seems that this function itself is getting rather larger

Reply:
Agree that we could break it out more.  The only thing is that I wanted to make sure that the fulfillments get created at the same time as the bounty because of the way that we do the `current_bounty`.  Maybe there is a cleaner way to do it...

Reply:
> Maybe there is a cleaner way to do it...

i think a subfunction that gets delegated to might do the trick.  could probably create a subfunction for the `interested` aboe functionality and the deletion (below) too.